raiden/tests/unit/rif_comms/test_rif_comms_client.py

```diff
@@ -1,181 +1,145 @@
-        assert subscribed is True
-
-    @pytest.mark.skip(reason="works but subscribed equals False")
+        # register node 1, locate node 2
+        response = self.client_1.connect()
+        assert self.client_1._get_peer_id(self.address_2) is ""
+
+    @pytest.mark.skip(reason="requires running rif comms node")
     def test_has_subscriber_self(self):
-        notification = self.rif_comms_client1.connect()
-        channel = self.rif_comms_client1.subscribe_to(LUMINO_1_ADDRESS)
-        subscribed = self.rif_comms_client1.is_subscribed_to(LUMINO_1_ADDRESS)
-        assert subscribed is True
+        # register node 1, subscribe to self, check subscription
+        notification = self.client_1.connect()
+        self.client_1.subscribe_to(self.address_1)
+        assert self.client_1.is_subscribed_to(self.address_1) is True
 
-    @pytest.mark.skip(reason="ignore")
+    @pytest.mark.skip(reason="hangs when attempting to sub 1 to 2 without subbing 2 to 2 before")
+    def test_has_subscriber(self):
+        # register nodes 1 and 2, subscribe from 1 to 2, check subscriptions
+        notification_1 = self.client_1.connect()
+        notification_2 = self.client_2.connect()
+        _, _ = self.client_1.subscribe_to(self.address_2)
+        assert self.client_1.is_subscribed_to(self.address_1) is False
+        assert self.client_1.is_subscribed_to(self.address_2) is True
+        assert self.client_2.is_subscribed_to(self.address_2) is False
+        assert self.client_2.is_subscribed_to(self.address_2) is False
+
+    @pytest.mark.skip(reason="hangs when attempting to sub 1 to 2 without subbing 2 to 2 before")
+    def test_two_clients_cross_subscription(self):
+        # register nodes 1 and 2
+        notification_1 = self.client_1.connect()
+        notification_2 = self.client_2.connect()
+
+        # subscribe to 1 and 2 on both nodes
+        self.client_1.subscribe_to(self.address_1)
+        self.client_1.subscribe_to(self.address_2)
+        self.client_2.subscribe_to(self.address_1)
+        self.client_2.subscribe_to(self.address_2)
+
+        assert self.client_1.is_subscribed_to(self.address_1) is True
+        assert self.client_1.is_subscribed_to(self.address_2) is True
+        assert self.client_2.is_subscribed_to(self.address_1) is True
+        assert self.client_2.is_subscribed_to(self.address_2) is True
+
+    @pytest.mark.skip(reason="incomplete test")
     def test_disconnect(self):
-        notification = self.rif_comms_client.connect()
-        peer_id = self.rif_comms_client._get_peer_id(LUMINO_1_ADDRESS)
-        self.rif_comms_client.disconnect()
+        # register node 1, get own peer id, disconnect
+        notification = self.client_1.connect()
+        peer_id = self.client_1._get_peer_id(self.address_1)
+        self.client_1.disconnect()
         # TODO check if end comms deletes topics
 
-    @pytest.mark.skip(reason="ignore")
-    def test_send_lumino_message(self):
```

Comment:
this test was meant to be temporary. 

it's already heavy enough to have to start up a comms node through python (which we plan to do soon); a lumino node on top of that is not worth the hassle.